devfiles/java-springboot/devfile.yaml

```diff
@@ -10,44 +10,37 @@ starterProjects:
 components:
   - name: tools
     container:
-      image: maysunfaisal/springbootbuild
+      image: quay.io/eclipse/che-java11-maven:nightly
       memoryLimit: 768Mi
-      command: ['tail']
-      args: [ '-f', '/dev/null']
       mountSources: true
-      volumeMounts:
-        - name: springbootpvc
-          path: /data
-  - name: runtime
-    container:
-      image: maysunfaisal/springbootruntime
-      memoryLimit: 768Mi
-      command: ['tail']
-      args: [ '-f', '/dev/null']
       endpoints:
-        - name: '8080/tcp'
-          targetPort: 8080
-      mountSources: false
+      - name: '8080-tcp'
+        targetPort: 8080
       volumeMounts:
-        - name: springbootpvc
-          path: /data
-  - name: springbootpvc
+        - name: m2
+          path: /home/user/.m2
+  - name: m2
     volume:
-      size: 4Gi
+      size: 3Gi
 commands:
-  - id: devBuild
+  - id: build
     exec:
       component: tools
-      commandLine: "/artifacts/bin/build-container-full.sh"
-      workingDir: /projects
+      commandLine: "mvn clean -Dmaven.repo.local=/home/user/.m2/repository package -Dmaven.test.skip=true"
```

Comment:
Why are the tests skipped?

Reply:
> do we need to run tests with build command?

It's debatable, indeed… 😄 
I might indeed not want the build to run the tests once I've pushed the component to the cluster since, presumably, these tests pass locally on my computer. Then again, I might want to run the tests on the cluster as well to check that my assumptions when running locally still hold on the cluster… This is a decent default as long as we make it clear that it can easily be changed by users.

Reply:
for tests shouldn't we encourage to use `odo test`?